src/code/codap-iframe-src.ts

```diff
@@ -25,7 +25,7 @@ const iframeSrc = (options: CodapParamsOptions) => {
 
   // assume release/branch names for params without slashes
   const expandBranchUrl = (url: string, branchUrl: string) => url.indexOf("/") === -1 ? branchUrl : url;
-  codap = expandBranchUrl(codap, `/releases/${codap}/static/dg/en/cert/`);
+  codap = expandBranchUrl(codap, `/releases/${codap}/static/dg/${lang}/cert/`);
```

Comment:
This change seems fine — this section might benefit from some examples of what can be passed in as acceptable values for `codap`.  At first glance it seems that there might be some overlap between what happens on line 23 and on line 28.